pulsar-broker/src/main/java/com/yahoo/pulsar/broker/admin/PersistentTopics.java

```diff
@@ -257,6 +257,23 @@
         }
     }
 
+    protected void validateAdminAndClientPermission(DestinationName destination) {
+        try {
+            validateAdminAccessOnProperty(destination.getProperty());
+        } catch (Exception ve) {
```

Comment:
> do you think it would be easy to not rely on exceptions 

We can but then we need to ignore code-modularity where we need to modify [property-check-function](https://github.com/yahoo/pulsar/blob/master/pulsar-broker/src/main/java/com/yahoo/pulsar/broker/web/PulsarWebResource.java#L169) which will validate permission on property and topic based on given booleans which will decide what to validate. If it will be ugly and if we don't touch this method then we need to add duplicate code to introduce such method. so, do you think it's worth to introduce duplicate-code or ignoring modularity and stats call may not be used that often in normal case.